repos/fdbt-site/src/pages/api/managePassengerTypes.ts

```diff
@@ -29,8 +80,19 @@ export const formatRequestBody = (req: NextApiRequestWithSession): [SinglePassen
         errors.push({ id: 'type', errorMessage: 'You must select a passenger type' });
     }
 
+    let isInEditMode = false;
+
+    const idIsANumber = Number.isInteger(id);
+
+    if (idIsANumber && id !== 0) {
```

Comment:
Is id=0 definitely not a valid case? Could we not go for undefined as unset?

Reply:
I believe it is because:

```
const emptyString = ""
console.log(Number(emptyString)); // logs 0
```